src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -109,6 +135,36 @@ public static void LogDependency(this ILogger logger, string dependencyType, obj
             logger.LogInformation(DependencyFormat, dependencyType, dependencyData, duration, startTime.ToString(CultureInfo.InvariantCulture), isSuccessful, context);
         }
 
+        /// <summary>
+        ///     Logs an Azure Service Bus Dependency.
+        /// </summary>
+        /// <param name="logger">Logger to use</param>
+        /// <param name="dependencyType">Type of the Service Bus entity</param>
+        /// <param name="isSuccessful">Indication whether or not the operation was successful</param>
+        /// <param name="measurement">Measuring the latency to call the Service Bus dependency</param>
+        /// <param name="context">Context that provides more insights on the dependency that was measured</param>
+        public static void LogServiceBusDependency(this ILogger logger, ServiceBusDependencyType dependencyType, bool isSuccessful, DependencyMeasurement measurement, Dictionary<string, object> context = null)
```

Comment:
Thinking out loud:
- Do we need to make the Service Bus dependency type optional? For example, do we always know if we are sending to a queue or topic?
- Instead of making it a parameter, what if it's just called `LogServiceBusQueueDependency`?

I would go for having a dedicated method per type (`LogServiceBusQueueDependency` & `LogServiceBusTopicDependency`) and provide `LogServiceBusDependency` which doesn't support it maybe?

Reply:
Yes, I can do that (the last proposal).

Reply:
Done that.